docs/content/en/docs/osmgmt/artifacts.md

```diff
@@ -929,6 +939,43 @@ These steps use `image-builder` to create a Ubuntu-based image for Nutanix AHV a
       image-builder build --os ubuntu --hypervisor nutanix --release-channel 1-25 --nutanix-config nutanix-connection.json
       ```
 
+
+### Configuring OS version
+
+`image-builder` supports an `os-version` flag that allows you to configure which version of the OS you wish to build. If no OS version is supplied, it will build the default for that OS, according to the table below
+
+|            | Supported versions | Default version |
+|:----------:|:-------------------:|:--------------:|
+| **Ubuntu** |     2004, 2204      |      2004      |
+|  **RHEL**  |          8          |       8        |
+
+Currently, Ubuntu is the only operating system that supports multiple versions.
```

Comment:
what about patch versions?

Reply:
Patch versions are determined by the corresponding Packer config file for the Hypervisor/OS combination we choose. For example, for Ubuntu 22.04 OVAs, the source image is an Ubuntu 22.04.1 ISO. https://github.com/kubernetes-sigs/image-builder/blob/main/images/capi/packer/ova/ubuntu-2204.json#L13

If upstream image-builder were to update this version and we pull in a new release of image-builder with this change, then future builds using image-builder will source the updated patch version of Ubuntu.

Reply:
Included patch versions in table.